app/components/osc-webhook-triggers/osc-webhook-triggers.html

```diff
@@ -1,38 +1,53 @@
-<ng-form name="secretsForm" class="add-webhook">
+<ng-form name="$ctrl.secretsForm" class="add-webhook">
   <div ng-repeat="trigger in $ctrl.webhookTriggers">
     <div class="add-webhook-row">
-      <ui-select 
-        ng-model="trigger.data.type"
-        ng-disabled="$ctrl.isDeprecated(trigger)"
-        on-select="$ctrl.triggerTypeChange(trigger)"
-        search-enabled="false"
-        title="Select a webhook type"
-        class="select-webhook-type"
-        flex>
-        <ui-select-match placeholder="Webhook type">
-          {{ $select.selected.label }}
-        </ui-select-match>
-        <ui-select-choices repeat="option.label as option in $ctrl.webhookTypesOptions">
-          {{ option.label }}
-        </ui-select-choices>
-      </ui-select>
+      <div class="select-webhook-type">
+        <ui-select 
+          ng-model="trigger.data.type"
+          name="triggerType{{$index}}"
+          ng-disabled="$ctrl.isDeprecated(trigger)"
```

Comment:
What happens if a trigger is deprecated? Is there a reason to notify the user?

Reply:
So webhook triggers that contain only plain inline secret, eg:
```
    - generic:
        secret: beb28e70e4b93d26
      type: Generic
```
Are deprecated and they cant be edited or added in the editor, just viewed and removed, Thats why both of their selectboxes are disabled.

Reply:
Ah, so would it be helpful to have a message telling the user "this is deprecated & cannot be edited"?